src/cli.rs

```diff
@@ -150,6 +152,24 @@ pub struct Cli {
         default_value = "#",
     )]
     pub topic: String,
+
+    /// Client cert path
+    #[arg(
+        long = "client-cert",
+        global = true,
+        env = "MQTTUI_CLIENT_CERTIFICATE",
+        value_hint = ValueHint::FilePath
+    )]
+    pub client_cert_path: Option<PathBuf>,
+
+    /// Client private key path
+    #[arg(
+        long = "client-key",
+        global = true,
+        env = "MQTTUI_CLIENT_PRIVATE_KEY",
+        value_hint = ValueHint::FilePath
+    )]
+    pub client_private_key: Option<PathBuf>,
```

Comment:
I think I would either rename the variable or the long attribute. Either its already clear in the short form then its that way both on CLI and in code or it needs to be more specific then it should be more specific both in code and CLI.

Also the current long form is not as specific as the env currently, this should also be aligned.

Maybe go with `client_key` and keep the doc comment saying its the private key?